src/goto-analyzer/static_verifier.h

```diff
@@ -36,4 +38,34 @@ void static_verifier(
   const ai_baset &ai,
   propertiest &properties);
 
+enum class ai_verifier_statust
+{
+  TRUE,
+  FALSE_IF_REACHABLE,
+  NOT_REACHABLE,
+  UNKNOWN
+};
+
+std::string as_string(const ai_verifier_statust &);
+
+/// The result of verifying a single assertion
+class static_verifier_resultt
+{
+public:
+  ai_verifier_statust status;
```

Comment:
❓ Should these members really be mutable?

Reply:
I mean... probably not but as `static_verifier_resultt` is really just a way to return a tuple of values I think it is better to just:

```
const static_verifier_resultt result(ai, property_location, "unused", ns);
```

and use them as `const` objects as is already done rather than write accesor functions.